### ISSUE TITLE
Wasm error when adding koin-compose-viewmodel artifact

### ISSUE THREAD

**Issue by BehnamMaboodi** [InsertKoinIO/koin#1946]
### Describe the Bug
Wasm error when adding the koin-compose-viewmodel artifact.

**To Reproduce**

Download the Compose Multiplatform template from [kmp.jetbrains.com](https://kmp.jetbrains.com/).
Add the following Koin dependencies to commonMain:
`koin-core`
`koin-compose`
`koin-compose-viewmodel`
The app works fine on iOS and Android, but on WebAssembly (Wasm), I get this error when I run the app:

```
ERROR
WebAssembly.instantiate(): Import #4340 "./skiko.mjs" "org_jetbrains_skia_Font__1nIsLinearMetrics": function import requires a callable
LinkError: WebAssembly.instantiate(): Import #4340 "./skiko.mjs" "org_jetbrains_skia_Font__1nIsLinearMetrics": function import requires a callable

```

When I remove koin-compose-viewmodel, the problem is resolved, so something is likely wrong with koin-compose-viewmodel.

### Koin Module and Version:
koin-compose-viewmodel:4.0.0-RC1

**Comment by Feinate** [comment]
Same here -> WebAssembly.instantiate(): Import #4379 "./skiko.mjs"

**Comment by amalhanaja** [comment]
Same here

Error Logs:

```
WebAssembly.instantiate(): Import #4337 "./skiko.mjs" "org_jetbrains_skia_Font__1nIsLinearMetrics": function import requires a callable
LinkError: WebAssembly.instantiate(): Import #4337 "./skiko.mjs" "org_jetbrains_skia_Font__1nIsLinearMetrics": function import requires a callable
```

is there any work around for this issue?

**Comment by amalhanaja** [comment]
Solved by upgrading compose plugin to version `1.7.0-alpha03`

**Comment by BehnamMaboodi** [comment]
Also solved in `4.0.0-RC2`
